src/sentry/deletions/__init__.py

```diff
@@ -21,7 +21,7 @@
 --------------------
 
 The entrypoint into deletions for the majority of application code is via the ``ScheduledDeletion``
-model. This model lets you creation deletion jobs that are run in the future.
+model. This model lets you create deletion jobs that are run in the future.
```

Comment:
No need to change this time around, but generally try to keep unrelated changes into separate prs

Reply:
Will do for next time!